frontend/src/services/eventHandlerService.js

```diff
@@ -1,18 +1,19 @@
-import { GlobalStore } from "@/modules/common/store/globalstore/index";
 import { EventBus } from "../eventBus.js";
+import EventHandlerFactory from "./eventHandlers/eventHandlerFactory";
 
+// Service for handling websocket events
 export default class EventHandlerService {
     handle(api, payload) {
-        if (api === "hub") {
-            this.handleHubEvent(payload);
+        console.log("----- Event Received -----");
+        console.log(api);
+        console.log(payload);
```

Comment:
make it so it doesn't log if it's api "draw"

Reply:
move console.log to print only for supported handlers